### PR TITLE
ch*: Remove duplicate declarations + start to plug it to chmod too

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by sylvestre** [comment]
before bigger changes

File: src/uu/chgrp/src/chgrp.rs [hunk 1]
```diff
@@ -140,24 +126,12 @@ pub fn uu_app() -> Command {
...
-                .action(ArgAction::SetTrue),
-        )
+        );
+
+    // Add common arguments with chgrp, chown & chmod
+    for arg in uucore::perms::common_args() {
+        cmd = cmd.arg(arg);
+    }
+
+    cmd
```
Comment (cakebaker) [inline src/uu/chgrp/src/chgrp.rs:136]: This is not necessary. `clap` provides an `args` function for adding multiple args and so you can use: `.args(uucore::perms::common_args())`. It also allows you to get rid of the `cmd` var.

File: src/uu/chmod/src/chmod.rs [hunk 2]
```diff
@@ -206,16 +214,24 @@ pub fn uu_app() -> Command {
...
                 .value_hint(clap::ValueHint::AnyPath),
-        )
+        );
+
+    // Add common arguments with chgrp, chown & chmod
+    for arg in uucore::perms::common_args() {
+        cmd = cmd.arg(arg);
+    }
+
+    cmd
```
Comment (cakebaker) [inline src/uu/chmod/src/chmod.rs:234]: See my other comment.

File: src/uu/chown/src/chown.rs [hunk 3]
```diff
@@ -165,34 +146,20 @@ pub fn uu_app() -> Command {
...
                 .action(ArgAction::SetTrue),
-        )
+        );
+
+    // Add common arguments with chgrp, chown & chmod
+    for arg in uucore::perms::common_args() {
+        cmd = cmd.arg(arg);
+    }
+
+    cmd
```
Comment (cakebaker) [inline src/uu/chown/src/chown.rs:162]: See my other comment.

File: src/uu/chgrp/src/chgrp.rs (hunk 1 above)
Comment (sylvestre) [reply src/uu/chgrp/src/chgrp.rs:136]: Much better, thanks :)

**Pull request merged**